packages/core/src/components/Badge/Badge.test.tsx

```diff
@@ -44,9 +44,7 @@ describe("Badge", () => {
       />
     );
     expect(screen.getByLabelText("99+")).toBeInTheDocument();
-    expect(
-      screen.getByRole("presentation", { name: "Alert" })
-    ).toBeInTheDocument();
+    expect(screen.getByLabelText("Alert")).toBeInTheDocument();
```

Comment:
We should fix the test `icon`, `role="presentation"` with `aria-label` isn't correct - we should make it `role="img"`
```suggestion
    expect(screen.getByRole("img", {name: "Alert"})).toBeInTheDocument();
```